server/etcdserver/server.go

```diff
@@ -1551,9 +1551,14 @@ func (s *EtcdServer) mayRemoveMember(id types.ID) error {
 	}
```

Comment:
If you unfold line 1548 to 1551, `mayRemoveMember` will get through when member id is downed already.

Reply:
Yes, I made a mistake: I actually tried to remove an up member, sorry for taking your time...

Reply:
No worries.